CONTRIBUTING.md

```diff
@@ -27,15 +27,6 @@ Adapted from the [Contributor Covenant] and [The Carpentries Code of Conduct]:
 With the current setup, the Nix manual hosted on nix.dev does not get updated automatically with new releases.
 The following manual steps are required:
 
-- Regularly update the inputs to use the latest versions of the Nix release branches with `nix shell --run "niv update"`
-
-  To avoid long build times, make sure Nix can be fetched from the cache.
-  If it doesn't, find the latest commit that is [built by Hydra](https://hydra.nixos.org/project/nix). For example, to pin Nix 2.18:
```

Comment:
This is kind of important, both because we don't want to render docs for branches that haven't been released yet, but also because we don't have any nix.dev cache that would cache such Nix builds for contributors.

Reply:
You're right, plus this is unrelated to the change I've made. I'll revert.